blockchain/types/accountkey/account_key.go

```diff
@@ -72,7 +72,7 @@ type AccountKey interface {
 	AccountCreationGas(currentBlockNumber uint64) (uint64, error)
 
 	// SigValidationGas returns gas required to validate a tx with the account.
-	SigValidationGas(currentBlockNumber uint64, r RoleType) (uint64, error)
+	SigValidationGas(currentBlockNumber uint64, r RoleType, sigNum int) (uint64, error)
```

Comment:
I prefer naming "num+plural" format for something's "number". In this case, `numSigs`
For me, `sigNum` sounds like a ID for a signature. e.g., sigNum=100 (it means 100 is signature's number)

Please check below example.

BlockNumber = A (unique) number assigned to a block
Number(Of)Blocks = How many blocks

As we need to merge this change today, please ignore above comment for now.
But please consider it in your later changes.

Reply:
Thinking back, it reads like that.. thanks!